storage/CMakeLists.txt

```diff
@@ -20,6 +20,8 @@ target_include_directories(sn_storage
     ${CMAKE_CURRENT_LIST_DIR}/src
 )
 
+add_subdirectory(../utils utils)
+target_link_libraries(sn_storage PRIVATE utils)
```

Comment:
after rebasing on master, you'll need to change to `loki_add_subdirectory` and change `sn_storage` to `storage`.